spec/models/member_spec.rb

```diff
@@ -31,11 +31,16 @@
           employed_pay_interval: "Hourly",
           employed_hours_per_week: 10,
         )
+        calc_double = double(run: true)
+        allow(MonthlyIncomeCalculator).to receive(:new).with(
+          pay_interval: "Hourly",
+          pay_quantity: 10,
+          hours_per_week: 10,
+        ).and_return(calc_double)
 
-        weekly_pay_times_average_weeks_per_month = 433.0
-        expect(member.monthly_income).to eq(
-          weekly_pay_times_average_weeks_per_month,
-        )
+        member.monthly_income
+
+        expect(calc_double).to have_received(:run)
       end
```

Comment:
The test says that "it returns monthly income", but we are asserting that `calc_double` has received `run`. Could we either:

1. Change the description to align with the test's expectation
2. Or, split this out into two tests where one tests the value of the monthly income, and the other tests that calc_double calls run?

Reply:
since we have comprehensive tests for the MonthlyCalculator itself, went with 1 (change description) at 251c58a 🎐